src/components/ProtestSelection/ProtestSelection.js

```diff
@@ -1,17 +1,26 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useState } from 'react';
 import { observer } from 'mobx-react-lite';
 import { useStore } from '../../stores';
 import styled from 'styled-components/macro';
 import { PlacesAutocomplete } from '../';
 import ProtestListSelection from './ProtestListSelection';
+import Loader from '../Loader';
 
 function ProtestSelection({ onProtestSelection, manualAddress = false }) {
   const store = useStore();
+  const [loadingProtests, setLoadingProtests] = useState(false);
```

Comment:
I think that `isLoadingProgress` will be better for this, since `loadingProtests` can be understood that it holds some protests data.